sass/bhclub.scss

```diff
@@ -25,6 +26,16 @@ a {
 h1 {
     font-size: 4.5rem;
     font-weight: normal;
+    padding-bottom: 3rem;
+}
+
+p {
+    font-size: 1.75rem;
+    padding-bottom: 1rem;
+}
+
+p:last-child {
+    padding-bottom: 0;
```

Comment:
Don't these changes conflict with changes made in [the contact section PR](https://github.com/codeRIT/brickhack.io/pull/1214/files#diff-dbdf174535c603db68638d757d8a27ced442ab27ef191daae26fcc303221f735R36)?

Reply:
Yes

I made the contact PR first, before I remembered that padding needs to be done in rem. The intention is to either merge that first and handle the merge conflicts with this or to merge this first and change it in the contact pr.